core/src/main/kotlin/org/evomaster/core/docs/ConfigToMarkdown.kt

```diff
@@ -180,8 +190,30 @@ object ConfigToMarkdown {
             buffer.append(" *Constraints*: `${description.constraints}`.")
         }
         if(description.enumValues.isNotBlank()){
-            buffer.append(" *Valid values*: `${description.enumValues}`.")
+            var experimentalValues = ""
```

Comment:
here `enumValues` is a string. likely we will need 2 separated strings, one for regular enum values, and another one for experimetal enum values (if any)